homeassistant/components/mqtt/mixins.py

```diff
@@ -588,6 +606,8 @@ class MqttEntity(
 ):
     """Representation of an MQTT entity."""
 
+    _entity_id_format = ""
```

Comment:
Just declare that this will be a `str` in a concrete class
```suggestion
    _entity_id_format: str
```

Reply:
done